src/components/henterInnhold/HenterInnhold.tsx

```diff
@@ -0,0 +1,25 @@
+import { Heading, Loader } from '@navikt/ds-react';
+import React from 'react';
+
+import { useI18n } from '~src/lib/i18n';
+
+import styles from './henterInnhold.module.less';
+
+const messages = {
+    'spinner.henterInnhold': 'Henter innhold...',
+};
```

Comment:
Enig at det er litt tiltak å legge denne i en egen fil, men tenker det uansett bør gjøres for å opprettholde samme struktur. Da vi egentlig skulle oversette alt til nynorsk, så hadde det vært mye enklere å finne tekstene når de er i egne filer og ikke sammen med komponentet :D

Veldig nice å trekke dette ut som et eget komponent, men kanskje vi skal kalle det noe mer generisk? Typ SpinnerMedTekst (som er et veldig dårlig navn) eller noe sånt? Så kan den placeholder-teksten heller være "Laster"

Reply:
Forresten, hvorfor er det en Loader med title, _og_ en Heading under?

Reply:
Da går vi for SpinnerMedTekst ;) 

den tittelen vises ikke noe sted. jeg antar det har noe med uu kanskje? 🤔